assets/scss/base/_typography.scss

```diff
@@ -0,0 +1,8 @@
+/* ==========================================================================
+Typography
+========================================================================== */
+
+body {
+    -webkit-font-smoothing: antialiased;
```

Comment:
I don't agree with applying font-smoothing by default since this setting depends explicitly on typeface weight and font-color contrast on background. This is effectively the same logic as applying a default `font-weight: 300`; of course you _could_ do it, but it would be better to leave that kind of setting up to the designer on a per project basis.

http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/